client/signup/steps/domains/index.jsx

```diff
@@ -812,7 +812,6 @@ class DomainsStep extends React.Component {
 				goToNextStep={ this.handleSkip }
 				skipHeadingText={ translate( 'Not sure yet?' ) }
 				skipLabelText={ translate( 'Choose a domain later' ) }
-				align={ isReskinned ? 'left' : 'center' }
```

Comment:
This is redundant (hence removed) since the alignment for both title & subtitle is defined in the css

Reply:
Reverted.